homeassistant/components/epson/__init__.py

```diff
@@ -3,32 +3,35 @@
 import logging
 
 from epson_projector import Projector
-from epson_projector.const import POWER, STATE_UNAVAILABLE as EPSON_STATE_UNAVAILABLE
+from epson_projector.const import (
+    PWR_OFF_STATE,
+    STATE_UNAVAILABLE as EPSON_STATE_UNAVAILABLE,
+)
 
 from homeassistant.components.media_player import DOMAIN as MEDIA_PLAYER_PLATFORM
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.const import CONF_HOST
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
 from .const import DOMAIN
-from .exceptions import CannotConnect
+from .exceptions import CannotConnect, PoweredOff
 
 PLATFORMS = [MEDIA_PLAYER_PLATFORM]
 
 _LOGGER = logging.getLogger(__name__)
 
 
-async def validate_projector(hass: HomeAssistant, host, port):
```

Comment:
I think we should keep the option to set HTTP port. Do all Epson projectors use the port 80? If we're not sure, keeping this option is a safe bet. We can make it optional.

Reply:
TCP connection can use 2 ports only 32xx. those ports cannot be changed.

Reply:
OK, and what about HTTP? Do all Epson projectors use the port 80 only? If so, then it makes sense to remove this option.

Reply:
HTTP is not official way at all of requesting data from Epson.
It was easier back than (3 yrs ago) for me to just sniff traffic from webview than try to implement protocol (which is also unofficial)

Reply:
Aha, OK... thanks for the info. I studied only the transport over RS232.